tests/src/lib.rs

```diff
@@ -21,40 +24,57 @@ mod tests {
         let prior_dot = std::fs::read_to_string(format!("{path}/road_network.dot"))
             .unwrap_or_else(|_| String::new());
 
-        let clip_pts = Some(LonLat::read_geojson_polygon(&format!(
-            "{path}/boundary.json"
-        ))?);
-        let (mut street_network, _) = streets_reader::osm_to_street_network(
-            &std::fs::read_to_string(format!("{path}/input.osm"))?,
-            clip_pts,
-            MapConfig::default(),
-            &mut timer,
-        )?;
+        let clip_pts = if Path::new(format!("{path}/boundary.json").as_str()).exists() {
```

Comment:
I like that we're testing this new logic, but the new test has a boundary file. Should we remove it?

Reply:
my vote would be to keep it. it means that all new pbf tests can optionally include a bounding box IF they want to, but it won't ever be required, since we can read the pbf header. that feels more flexible to me, but it's of course your call !